community/common/src/main/java/org/neo4j/kernel/api/exceptions/Status.java

```diff
@@ -482,7 +482,9 @@ public Code code()
         NoLeader( TransientError,
                 "No leader available at the moment. Retrying your request at a later time may succeed." ),
 
-        ;
+        NotLeader( TransientError,
```

Comment:
Given the previous status being named `NoLeader`, it may be confusing with the `NotLeader` name. Can we rename `NoLeader` to `NoLeaderAvailable` and `NotLeader` to `NotALeader`